output/eu_crude/analysis.py

```diff
@@ -0,0 +1,152 @@
+import sqlite3 as lite
+import sys
+from pyne import nucname
+
+# tells command format if input is invalid
+if len(sys.argv) < 2:
+    print('Usage: python analysis.py [cylus_output_file]')
+
+
+def snf(filename):
+    """ does simple analysis of cyclus input file.
```

Comment:
simple analysis could mean anything. Can you be more specific? What, exactly, does this function do.